gp-ant-task/example/build.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<project name="gp-ant-task-1.1.5-SNAPSHOT" default="download-translations" 
+	xmlns:gp="antlib:com.ibm.g11n.pipeline.ant">
+	
+	<taskdef  uri="antlib:com.ibm.g11n.pipeline.ant" resource="com/ibm/g11n/pipeline/ant/antlib.xml">
+		<!-- If the gp-ant-task jar is not added to ANT_HOME/lib folder, 
+			then you may include the classpath for the jar like in the comment below-->
+		<!--<classpath path="lib/gp-ant-task-1.1.5-SNAPSHOT-with-dependencies.jar"/>-->
+	</taskdef>
+	
+	<!--Specify the location of the credentials json file-->
+	<property name="gp.credentials" value="src/credentials.json"/>
+	
+	<!-- basic usecase for uploading resources(this uploads java .properties files by default)-->
+    <target name="upload-resources">
+        <gp:upload credentialsJson= "${gp.credentials}" sourceDir="src/main/resources">
+        </gp:upload>
```

Comment:
Can we specify this in a single line as below?
<gp:upload credentialsJson= "${gp.credentials}" sourceDir="src/main/resources" />